database-commons/src/main/java/co/cask/util/DBUtils.java

```diff
@@ -327,6 +332,22 @@ public static void deregisterAllDrivers(Class<? extends Driver> driverClass)
     }
   }
 
+  public static void validateJDBCPluginPipeline(PipelineConfigurer pipelineConfigurer,
+                                                ConnectionConfig config, String jdbcPluginId, String jdbcDriverName) {
+    Preconditions.checkArgument(!(config.user == null && config.password != null),
```

Comment:
in CDAP 6.0.0, we added an InvalidConfigPropertyException that should be thrown when a plugin is performing validation on its config properties and notices that one is invalid. Please change the code so that it is used instead of IllegalArgumentException